toolchain/lower/file_context.cpp

```diff
@@ -512,7 +511,8 @@ static auto BuildTypeForInst(FileContext& context, InstT /*inst*/)
 // Treat non-monomorphized symbolic types as opaque.
 template <typename InstT>
   requires(InstT::Kind.template IsAnyOf<SemIR::BindSymbolicName,
-                                        SemIR::InterfaceWitnessAccess>())
+                                        SemIR::InterfaceWitnessAccess,
+                                        SemIR::SymbolicBindingPattern>())
```

Comment:
Yes. I think this another instance of the point discussed elsewhere, about using patterns as values.

Reply:
I think this case is a little different. Even if `SymboilcBindingPattern` can be used as a value, it shouldn't be showing up here: it can't be an instruction referred to by a `TypeId`, because it constant evaluates to a `BindSymbolicName` not to a `SymbolicBindingPattern`. I think its `InstKind` should classify it as `InstIsType::Never` because it's never the canonical definition of a type.

Reply:
OK, that seems to work.